frontend/benefit/applicant/src/components/mainIngress/MainIngress.tsx

```diff
@@ -0,0 +1,39 @@
+import { IconPlus } from 'hds-react';
+import * as React from 'react';
+
+import { useComponent } from './extended';
+import {
+  StyledActionContainer,
+  StyledButton,
+  StyledContainer,
+  StyledDescription,
+  StyledHeading,
+  StyledLink,
+  StyledTextContainer,
+} from './styled';
+
+const MainIngress: React.FC = () => {
+  const { handleMoreInfoClick, t } = useComponent();
+
+  return (
+    <StyledContainer>
```

Comment:
nitpick also but this Styled gets quite repetitive, could ir be just <Container> and same with others? I see that it's a convention to separate custom components from simply styled components, but still i find it quite repetitive. I've been a project that used StyledComponents but we didnt prefix the components this way. I'm not sure which one is good way to go :)

Reply:
Ah, thats basically the way to quickly separate them from other components. so you see what is styled and what is just custom ones, especially when the structure is complex it might be helpful

Reply:
It's a purely matter of taste so you can keep it this "style" if you want :)